packages/experiments/src/components/UnifiedPicker/UnifiedPicker.tsx

```diff
@@ -8,8 +8,7 @@ import {
   FocusZone,
   SelectionZone,
   Autofill,
-  IInputProps,
-  MarqueeSelection,
+in  IInputProps,
```

Comment:
extra in, looks like a typo

Reply:
Yes, saw that an fixed now.  Focus issues.